src/libs/OptionsListUtils.js

```diff
@@ -662,10 +692,26 @@ function getCategoryOptionTree(options, isOneLine = false) {
  */
 function getCategoryListSections(categories, recentlyUsedCategories, selectedOptions, searchInputValue, maxRecentReportsToShow) {
     const categorySections = [];
-    const categoriesValues = _.values(categories);
+    const categoriesValues = _.chain(categories)
```

Comment:
I think is more readable like:
```
Object.values(categories).filter((category) => category.enabled);
```

Reply:
We still has a lint rule for `.js` files.
<img width="668" alt="Screenshot 2023-09-19 at 11 30 37" src="https://github.com/Expensify/App/assets/57314631/a7c9a97f-2b87-4f8c-bcd8-8f6a258c2e1b">

Reply:
However, I agree to simplify the logic. Let me try to play with it.

Reply:
The simplest way is using `chain` for now here.